homeassistant/components/light/limitlessled.py

```diff
@@ -33,32 +37,54 @@
 from homeassistant.util.color import color_RGB_to_xy
 
 _LOGGER = logging.getLogger(__name__)
-REQUIREMENTS = ['ledcontroller==1.0.7']
+REQUIREMENTS = ['ledcontroller==1.1.0']
 
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the LimitlessLED lights. """
     import ledcontroller
 
-    led = ledcontroller.LedController(config['host'])
+    # Handle old configuration format:
+    bridges = []
```

Comment:
Also, changing config is fine, especially if we can offer a backwards compatible config.

It was already possible to specify multiple hosts like this:
```yaml
light:
  platform: limitlessled
  host: …

light 2:
  platform: limitlessled
  host: …
```
But that would have prevented us from using a `LedControllerPool` which seems to be a better approach :+1: